packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -1169,35 +1149,31 @@ exports[`Podcast Promo card-call-to-action should match snapshot 1`] = `
 
 <div>
   <span
-    aria-hidden="true"
+    class="emotion-2 emotion-3"
   >
-    <i
-      class="emotion-2 emotion-3"
+    <svg
+      aria-hidden="true"
+      class="emotion-0 emotion-1"
+      focusable="false"
+      height="32"
+      viewBox="0 0 32 32"
+      width="32"
     >
-      <svg
-        aria-hidden="true"
-        class="emotion-0 emotion-1"
-        focusable="false"
-        height="32"
-        viewBox="0 0 32 32"
-        width="32"
-      >
-        <polygon
-          points="4 6 11.1 6 26 6 26 28 28 28 28 4 4 4 4 6"
-        />
-        <polygon
-          points="8 0 8 2 30 2 30 24 32 24 32 0 8 0"
-        />
-        <path
-          d="M0,32H24V8H0ZM4,12H20V28H4Z"
-        />
-      </svg>
-    </i>
-    <span
-      class="emotion-4 emotion-5"
-    >
-      Click Me
-    </span>
+      <polygon
+        points="4 6 11.1 6 26 6 26 28 28 28 28 4 4 4 4 6"
+      />
+      <polygon
+        points="8 0 8 2 30 2 30 24 32 24 32 0 8 0"
+      />
+      <path
+        d="M0,32H24V8H0ZM4,12H20V28H4Z"
+      />
+    </svg>
   </span>
+  <p
+    class="emotion-4 emotion-5"
+  >
+    Click Me
```

Comment:
Please could we use better text, this isn't a link.

Reply:
Changed to 'Episodes'